components/content-service/cmd/run.go

```diff
@@ -32,106 +19,63 @@ var runCmd = &cobra.Command{
 
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg := getConfig()
-		reg := prometheus.NewRegistry()
 
-		common_grpc.SetupLogging()
+		port, err := cfg.Service.Port()
+		if err != nil {
+			log.WithError(err).Fatal("Failed to get service port.")
+		}
 
-		grpcMetrics := grpc_prometheus.NewServerMetrics()
-		grpcMetrics.EnableHandlingTimeHistogram()
-		reg.MustRegister(grpcMetrics)
+		opts := []baseserver.Option{
+			baseserver.WithGRPCPort(port),
```

Comment:
Would this start the prometheus/debug endpoint, too?

Reply:
Once https://github.com/gitpod-io/gitpod/pull/9742 lands, then every base-server will always have metrics, pprof, health and ready endpoints. We'll need to do some port standardization as well to drive more consistency but with that, this will have it out of the box so no need for custom config.